src/index.js

```diff
@@ -974,13 +958,10 @@ export default class Carousel extends React.Component {
         slidesToScroll,
         slidesToShow,
         slideWidth,
-        cellAlign,
-        left: props.vertical ? 0 : this.getTargetLeft(),
-        top: props.vertical ? this.getTargetLeft() : 0
+        cellAlign
       },
       () => {
         stateCb();
-        this.setLeft();
```

Comment:
Removing the call to `setLeft` and the updates to `left` and `top` from the method `setDimensions`, fixed the wrap around problem we saw when `heightMode` was set to `current`.  I beliiieeeeve (but I'm not positive), that these extra calls to set the `left` and `top`, was just paranoiac ways to ensure that the carousel wouldn't blink.  But, I think the new wrapAround logic should prevent it sooooooooo... I don't think we need these calls.  I've been playing around with the demo a bit and it all still works.